crates/hdk/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- hdk: Provide `Into<AnyHashable>` impl for `EntryHash` and `HeaderHash`. This allows `create_link` and `get_links` to be used directly with EntryHash and HeaderHash arguments, rather than needing to construct an AnyHashable explicitly.
```

Comment:
```suggestion
- hdk: Provide `Into<AnyHashableHash>` impl for `EntryHash` and `HeaderHash`. This allows `create_link` and `get_links` to be used directly with EntryHash and HeaderHash arguments, rather than needing to construct an AnyHashable explicitly.
```

because i couldn't find the `AnyHashable` anywhere